test/metrics/metrics.ts

```diff
@@ -209,6 +209,7 @@ describe('Test metrics with mock server', () => {
       spannerMock.resetRequests();
       spannerMock.removeExecutionTimes();
       // Reset the MetricsFactoryReader to an in-memory reader for the tests
+      MetricsTracerFactory.enabled = true;
```

Comment:
why are enabling it here now? we already are enabling it to true in before block of parent

Reply:
because in after each block we are resetting MetricsTracerFactory, earlier it was always default enabled , but now with mockSpanner since it is using inSecureCredentials MetricsTracerFactory.enabled is false.